databricks/koalas/namespace.py

```diff
@@ -1834,9 +1843,11 @@ def resolve_func(kdf, this_column_labels, that_column_labels):
     # DataFrame, Series ... & Series, Series ...
```

Comment:
Maybe we should throw an exception when `axis=1` and `sort` is set:

```python
>>> import pandas as pd
>>> pd.concat([pd.DataFrame({'b': [4, 5, 6]}, index=[4,3,2]), pd.DataFrame({'a': [3, 4, 5]}, index=[4,3,2])], sort=False, axis=1)
   b  a
4  4  3
3  5  4
2  6  5
>>> pd.concat([pd.DataFrame({'b': [4, 5, 6]}, index=[4,3,2]), pd.DataFrame({'a': [3, 4, 5]}, index=[4,3,2])], sort=True, axis=1)
   b  a
2  6  5
3  5  4
4  4  3
```

Reply:
or do `sort_index()`?

Reply:
Anyway pandas' behavior is not deterministic..

Reply:
😭 .. anyway is fine. We could just `sort_index()` or just throw an exception